### PR TITLE
Allow forcing of a specific language through env file

### PR DESCRIPTION
**Pull request by rudivanhierden**
Use the FORCE_LANGUAGE key in the env file to force a specific language, e.g. FORCE_LANGUAGE=nl

Related ticket: https://trello.com/c/jUTokk7r/649-vertalingen-mogelijkheid-om-%C3%A9%C3%A9n-taal-te-forceren

### CODE REVIEW

File: siteConfig.js [hunk 1]
```diff
@@ -281,10 +281,19 @@ module.exports = {
       siteConfig.modules['apostrophe-workflow-modified-documents'] = {};
 
     } else {
+      
+      let locales = ['nl', 'en'];
```
Comment (stijnvandervegt) [inline siteConfig.js]: I don't like `let` better use `const` :-)

File: siteConfig.js [hunk 2]
```diff
@@ -281,10 +281,19 @@ module.exports = {
...
+      let defaultLocale = 'nl';
+      
+      if (process.env.FORCE_LANGUAGE) {
+        locales = [process.env.FORCE_LANGUAGE];
```
Comment (stijnvandervegt) [inline siteConfig.js]: Missing some documentation about this env variable `FORCE_LANGUAGE`

Comment (stijnvandervegt) [reply siteConfig.js]: Or we have to introduce a .env.example file

**Comment by rudivanhierden** [comment]
PR made

**Pull request closed without merging**